hs_core/hydroshare/hs_bagit.py

```diff
@@ -128,20 +119,32 @@ def create_bag(resource):
 
     # change the namespace for the 'creator' element from 'dcterms' to 'dc'
     xml_string = remdoc.data.replace('dcterms:creator', 'dc:creator')
-    with open(bagit_path + '/resourcemap.xml', 'w') as out:
+
+    # create resourcemap.xml and upload it to iRODS
+    from_file_name = '{path}/resourcemap.xml'.format(path=bagit_path)
+    with open(from_file_name, 'w') as out:
         out.write(xml_string)
+    to_file_name = '{res_id}/data/resourcemap.xml'.format(res_id=resource.short_id)
+    istorage.saveFile(from_file_name, to_file_name, False)
+
+    # call iRODS bagit rule here
+    irods_dest_prefix = "/" + settings.IRODS_ZONE + "/home/" + settings.IRODS_USERNAME
+    irods_bagit_input_path = os.path.join(irods_dest_prefix, resource.short_id)
+    bagit_input = "*BAGITDATA='{path}'".format(path=irods_bagit_input_path)
+    bagit_rule_file = getattr(settings, 'IRODS_BAGIT_RULE', 'hydroshare/irods/ruleGenerateBagIt_HS.r')
```

Comment:
I put this setting in local_settings.py and thought it won't hurt to add a default here just in case someone has an old local_settings.py. Thought it is also good to indicate a default value here. The settings will override this default value anyway, so I don't see any harm to put a default here.  Let me know if there are strong reasons to remove default here.